src/rule_tester.rs

```diff
@@ -38,18 +47,39 @@ impl RuleTester {
         {
             panic!("Specified 'output' for a non-fixable rule");
         }
-        let languages = rule.meta().languages;
-        if languages.len() != 1 {
-            panic!("Only supporting single-language rules currently");
-        }
         Self {
-            language: languages[0],
+            supported_language_languages: rule
+                .meta()
+                .languages
+                .iter()
+                .flat_map(|language| {
+                    language
+                        .all_supported_language_languages()
+                        .into_iter()
+                        .copied()
+                })
+                .collect(),
             rule,
             rule_tests,
             from_file_run_context_instance_provider_factory,
+            plugins,
+            should_aggregate_results: env::var("RULE_TEST_SUMMARY").ok().is_non_empty(),
```

Comment:
This is a nice feature for showing all passing/failing cases (similar to what ESLint shows) vs just failing on first failing case